src/app/api/history/route.ts

```
@@ -0,0 +1,53 @@
+    const stats = await query.getRawMany()
+    console.log(stats);
+    return NextResponse.json({
+      stats
+    });
+  } catch (error) {
+    console.error("Error fetching deals by stage:", error);
+    return NextResponse.json(
+      { error: "Internal server error" },
+      { status: 500 }
+    );
+  };
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)